Preselect last -usually plasma-wayland- instead of first, usually any other X11 session.

Alternatively patch default state (/var/lib/plasmalogin/state.conf) if needed.

--- plasma-login-manager/src/frontend/greeter/qml/SessionButton.qml	2000-01-01 01:00:00.000000000 +0100
+++ plasma-login-manager.2/src/frontend/greeter/qml/SessionButton.qml	2026-01-20 20:28:35.330983228 +0100
@@ -34,7 +34,7 @@
         } else if (lastLoggedInSessionIndex != -1) {
             currentIndex = lastLoggedInSessionIndex;
         } else {
-            currentIndex = 0;
+            currentIndex = instantiator.count > 0 ? instantiator.count - 1 : 0;
         }
     }
 
